django/contrib/staticfiles/storage.py

```diff
@@ -473,20 +471,6 @@ def hash_key(self, name):
         return 'staticfiles:%s' % key
 
 
-class CachedStaticFilesStorage(CachedFilesMixin, StaticFilesStorage):
```

Comment:
I think that we should also remove `CachedFilesMixin` and `_MappingCache` that are currently unused. (\cc @carltongibson).

Reply:
Yes, absolutely.